DocumentDataAPI/DocumentDataAPI/Data/Repositories/NpgDocumentRepository.cs

```diff
@@ -53,10 +53,10 @@ public async Task<IEnumerable<DocumentModel>> GetAll(DocumentSearchParameters pa
         {
             QueryParameter firstParam = parameters.Parameters.First();
             args.Add(firstParam.Key, firstParam.Value);
-            query.Append($" where {firstParam.Key} {firstParam.ComparisonOperator} @{firstParam.Key}");
+            query.Append($" where " + _sqlHelper.GetParameterString(firstParam));
```

Comment:
string interpolation er vel ikke nødvendigt længere.
```suggestion
            query.Append(" where " + _sqlHelper.GetParameterString(firstParam));
```

Reply:
avada kedavra